README.md

```diff
@@ -0,0 +1,79 @@
+# Environment Variables Injector
```

Comment:
```suggestion
# The env-injector extension
```

Consistent with other extension docs: https://snapcraft.io/docs/supported-extensions

Reply:
Yeah, that makes sense! Thank's!